pkg/plugin/plugin.go

```diff
@@ -136,3 +139,24 @@ func (p *plugin) CreateOrUpdate(ctx context.Context, cs *api.OpenShiftManagedClu
 	// explicitly return nil if all went well
 	return nil
 }
+
+func (p *plugin) ClusterStatus(ctx context.Context, cs *api.OpenShiftManagedCluster) ([]byte, error) {
```

Comment:
Do we want to get logs here or from a different endpoint?

Reply:
I think logs will be from another endpoint...if we actually want that. I personally don't think its a good idea. I'd prefer a portal of sorts for where logs are aggregated.